Tests/ReactorKitTests/ReactorTests.swift

```diff
@@ -453,3 +502,63 @@ private final class CounterReactor: Reactor {
     return state + 1
   }
 }
+
+final class TestCounterReactor: Reactor {
```

Comment:
```suggestion
private final class TestCounterReactor: Reactor {
```

Please use the **private** access control
so that it can only be accessed from ReactorTests.swift file.

Reply:
Oh, thank you! :)